src/DssSpell.t.sol

```diff
@@ -51,152 +40,60 @@ contract DssSpellTest is DssSpellTestBase {
-        }
 
-        // We have failed if we reach here
-        assertTrue(false, "TestError/GiveTokens-slot-not-found");
     }
-
-    function checkPsmIlkIntegrationGUSD(
-        bytes32 _ilk,
-        GemJoinAbstract join,
-        ClipAbstract clip,
-        address pip,
-        PsmAbstract psm,
-        uint256 tin,
-        uint256 tout
-    ) public {
-        DSTokenAbstract token = DSTokenAbstract(join.gem());
-
-        assertTrue(pip != address(0));
-
-        spotter.poke(_ilk);
-
-        // Authorization
-        assertEq(join.wards(pauseProxy), 1);
-        assertEq(join.wards(address(psm)), 1);
-        assertEq(psm.wards(pauseProxy), 1);
-        assertEq(vat.wards(address(join)), 1);
-        assertEq(clip.wards(address(end)), 1);
-
-        // Check toll in/out
-        assertEq(psm.tin(), tin);
-        assertEq(psm.tout(), tout);
-
-        uint256 amount = 1000 * (10 ** token.decimals());
-        giveTokensGUSD(token, amount);
-
-        // Approvals
-        token.approve(address(join), amount);
-        dai.approve(address(psm), uint256(-1));
-
-        // Convert all TOKEN to DAI
-        psm.sellGem(address(this), amount);
-        amount -= amount * tin / WAD;
-        assertEq(token.balanceOf(address(this)), 0);
-        assertEq(dai.balanceOf(address(this)), amount * (10 ** (18 - token.decimals())));
-
-        // Convert all DAI to TOKEN
-        amount -= amount * tout / WAD;
-        psm.buyGem(address(this), amount);
-        assertEq(dai.balanceOf(address(this)), 0);
-        assertEq(token.balanceOf(address(this)), amount);
-
-        // Dump all dai for next run
-        vat.move(address(this), address(0x0), vat.dai(address(this)));
-    }
-
     function testCollateralIntegrations() public {
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new collateral tests here
         checkIlkIntegration(
-            "WBTC-C",
-            GemJoinAbstract(addr.addr("MCD_JOIN_WBTC_C")),
-            ClipAbstract(addr.addr("MCD_CLIP_WBTC_C")),
-            addr.addr("PIP_WBTC"),
-            true,
+            "GUNIV3DAIUSDC2-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_GUNIV3DAIUSDC2_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_GUNIV3DAIUSDC2_A")),
+            addr.addr("PIP_GUNIV3DAIUSDC2"),
             true,
+            false,
             false
         );
-        checkPsmIlkIntegrationGUSD(
-            "PSM-GUSD-A",
-            GemJoinAbstract(addr.addr("MCD_JOIN_PSM_GUSD_A")),
-            ClipAbstract(addr.addr("MCD_CLIP_PSM_GUSD_A")),
-            addr.addr("PIP_GUSD"),
-            PsmAbstract(addr.addr("MCD_PSM_GUSD_A")),
-            0,
-            0
-        );
     }
+       
+    
 
-    function testLerpSurplusBuffer() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+    // function testLerpSurplusBuffer() public {
```

Comment:
remove or move to the test base

Reply:
lets move to test base, but I'm happy to add that to my spell clean up task so we don't add diffs here